### PR TITLE
Added docker-compose app version

### PR DESCRIPTION
**Pull request by wolfy-j**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,45 @@ The samples in this repository demonstrate the various capabilities of the [Temp
...
+$ docker-compose exec app php app.php {sample-name}
+```
+
+To observe active workers:
+
+```bash
+$ docker-compose exec app rr workers -i
+```
+
+## Local Setup
```
Comment (flossypurse) [inline README.md:38]: What is the difference between Docker Compose Setup and Local Setup?

Seems like there are now two different approaches to running these sample?

Comment (wolfy-j) [reply README.md:38]: Docker setup basically one file to run. Local - install PHP, GRPC and etc. So it's a bit harder. I would presume we can easily remove the info about local setup and keep only Docker one.

Comment (flossypurse) [reply README.md:38]: I think having both paths is totally cool! 
I suggest we just call out this difference above -

File: README.md [hunk 2]
```diff
@@ -4,15 +4,45 @@ The samples in this repository demonstrate the various capabilities of the [Temp
 
 If you want to learn more about the Temporal Server and how it works, read the documentation at [https://docs.temporal.io](https://docs.temporal.io).
```
Comment (flossypurse) [inline README.md:6]: May call out the difference between Docker Compose setup and Local setup here -

Comment (wolfy-j) [reply README.md:6]: I can add more info tomorrow unless you'll do it.

File: README.md [hunk 3]
```diff
@@ -48,14 +83,19 @@ Note: You can install RoadRunner manually by downloading its binary from the [re
 The Temporal Server must be up and running for the samples to work.
 The fastest way to do that is by following the [Quick install guide](https://docs.temporal.io/docs/server-quick-install).
 
+You can also run the included `docker-compose.yml` file. Make sure to comment `app` section.
+
+**7. Update configuration**
+Make sure to update the temporal address in `app/.rr.yaml` to `localhost:7233`.  
+
 **7. Start the application using RoadRunner**
```
Comment (flossypurse) [inline README.md:91]: bump this to 8

File: README.md (hunk 1 above)
Comment (wolfy-j) [reply README.md:43]: i added info about the core differences, feel free to change it

**Review by flossypurse — APPROVED** [review]

**Pull request merged**